boards/ek-tm4c1294xl/Makefile

```diff
@@ -0,0 +1,31 @@
+# Makefile for building the tock kernel for the EK-TM4C1294XL platform
+
+TOCK_ARCH=cortex-m4
+TARGET=thumbv7em-none-eabihf
```

Comment:
wow, does `floating point` work out of the box in Tock?

I guess, we should change `nrf52` too if that is the case!

Reply:
Hrm... it _should_ because you need to enable it explicitly in the hardware and then the interrupt enter/exit sequence changes...

Reply:
Though enabling it for NRF52 might be a good test :)